Documentation/network/bgp-control-plane-operation.rst

```diff
@@ -68,6 +68,36 @@ forwarded to a different node than before, will be reset.
 
 .. _Resilient Hashing: https://www.juniper.net/documentation/us/en/software/junos/interfaces-ethernet-switches/topics/topic-map/switches-interface-resilient-hashing.html
 
+Node Down
+---------
+
+If the node goes down, the BGP session will be lost, and the Cilium datapath
+forwarding will be stopped. If the node is shut down gracefully with a proper
+BGP shutdown procedure or TCP connection close, the BGP peer either stops
```

Comment:
Should we document what is the `proper BGP shutdown procedure` or graceful node shutdown mechanism ?

Reply:
I added a new `Node Shutdown` section. Please take a look.